src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/src/Language/Syntax/GreenNode.cs

```diff
@@ -72,7 +74,7 @@ protected void AdjustFlagsAndWidth(GreenNode node)
         }
 
         Flags |= (node.Flags & NodeFlags.InheritMask);
```

Comment:
I wonder if `NodeFlags.ContainsStructuredTrivia` should be also removed.

Reply:
I was thinking of that as well. I performed a Find All References on each of the `NodeFlags` members and only a couple are actually used.